flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/CollectionFunctionsITCase.java

```diff
@@ -102,6 +105,16 @@ Stream<TestSetSpec> getTestSetSpecs() {
                                 "ARRAY_CONTAINS(f4, NULL)",
                                 true,
                                 DataTypes.BOOLEAN().nullable())
+                        .testResult(
+                                $("f5").arrayContains(lit(null, DataTypes.INT())),
+                                "ARRAY_CONTAINS(f5, CAST(NULL AS INT))",
+                                false,
+                                DataTypes.BOOLEAN().notNull())
+                        .testResult(
+                                $("f5").arrayContains(lit(4, DataTypes.INT().notNull())),
+                                "ARRAY_CONTAINS(f5, 4)",
+                                false,
+                                DataTypes.BOOLEAN().notNull())
```

Comment:
would be nice to have a test checking for containing `null` with `true` as expected result

Reply:
added